src/lib/mina_base/parties.ml

```diff
@@ -460,6 +460,14 @@ let inner_query =
     (Option.value_exn ~message:"Invariant: All projectable derivers are Some"
        Fields_derivers_snapps.(inner_query (deriver @@ Derivers.o ())))
 
+let other_parties_deriver_from_json json =
```

Comment:
can you move this up next to `of_json` and `to_json` and then name it `other_parties_of_json`?